x-pack/plugins/security_solution/public/exceptions/routes.tsx

```diff
@@ -23,7 +23,7 @@ import { useReadonlyHeader } from '../use_readonly_header';
 import { PluginTemplateWrapper } from '../common/components/plugin_template_wrapper';
 
 const ExceptionsRoutes = () => (
-  <PluginTemplateWrapper>
+  <PluginTemplateWrapper paddingSize="none" restrictWidth={false}>
```

Comment:
Looks like, setting this value to None, broke exceptions layout

### Before

<img width="2545" alt="Screenshot 2023-06-22 at 13 08 43" src="https://github.com/elastic/kibana/assets/92328789/1aedd9d1-8818-4f2a-98ca-09b1d55e3ec1">

### After

<img width="2510" alt="Screenshot 2023-06-22 at 13 09 08" src="https://github.com/elastic/kibana/assets/92328789/55e83535-b53b-49f6-9ee1-4b6f2403a474">

Reply:
@vitaliidm Thanks for catching this. I've reverted the change. Rules exception does not render GetStartedPageComponent when no indices exists, so it shouldn't need `paddingSize="none" restrictWidth={false}` on PluginTemplateWrapper.

<img width="2549" alt="Screenshot 2023-06-22 at 14 11 25" src="https://github.com/elastic/kibana/assets/6295984/503b0ad9-bb6d-46fa-9237-bf876593a688">

<img width="2549" alt="Screenshot 2023-06-22 at 14 15 47" src="https://github.com/elastic/kibana/assets/6295984/7c03f09e-b2a2-49d4-bc83-5adfb531e93e">